src/components/auth/SignIn.js

```diff
@@ -0,0 +1,79 @@
+import "./SignIn.css";
+import circle_sprite from "../../assets/circle_sprite.png";
+import scaleneTriangle_sprite from "../../assets/scaleneTriangle_sprite.png";
+import {getAuth, signInWithEmailAndPassword} from "firebase/auth";
+import {useState} from "react"
+
+
+const SignInScreen = ({firebaseApp}) => {
+  // takes the entered email and password and logs in the user
+  const auth = getAuth(firebaseApp);
+
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value)
+  }
+
+  const handlePasswordChange = (e) => {
+    setPassword(e.target.value)
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    signInWithEmailAndPassword(auth, email, password)
+    .then((userCredential) => {
+      const user = userCredential.user;
+      window.location.href = "/";
+      console.log("success!");
```

Comment:
typically, we try to remove `console.log`. remove in next go around?

Reply:
Done